assets/js/pluginsidebar/sidebar.jsx

```diff
@@ -118,7 +118,10 @@ const Sidebar = () => {
    */
   const displayNotification = useCallback((message, type = 'success') => (type === 'success'
     ? dispatchNotice.createInfoNotice(DOMPurify.sanitize(message), { type: 'snackbar' })
-    : dispatchNotice.createErrorNotice(DOMPurify.sanitize(message))
+    : dispatchNotice.createErrorNotice(
+      DOMPurify.sanitize(message, { ADD_TAGS: ['br'] }),
```

Comment:
Why is `ADD_TAGS` necessary here? It should be able to output the same HTML that was run through `wp_kses_post` without modification.

Reply:
Good point.  No reason to re-sanitize if it's already been done on the backend.